boot/environ/app/rosload/rosload.c

```diff
@@ -882,7 +882,7 @@ ArchRestoreProcessorFeatures (
     if (ArchXCr0BitsToClear)
     {
         /* Clear them */
-#if defined(_MSC_VER) && !defined(__clang__)
+#if defined(_MSC_VER) && !defined(__clang__) && !defined(_M_ARM)
```

Comment:
And otherwise? I mean it was like this before, but shouldn't we handle the alternative case? Maybe with an #error?

Reply:
We probably should, but I don't think we will use this on anything other than x86 / x64 anyway, since most code is written for that.